### ISSUE TITLE
Refresh token expired

### ISSUE THREAD

**Issue by KingKarlo** [hugot/n2wal#1]
First of all, thank you for the great tool! 

It helps perfectly to keep all the places where I read news (Miniflux and Wallabag) in sync.

One question and one error I've encountered:

1. Could it be that n2wal is syncing "only" 100 items per feed? I have feeds with more than 100 items and it looks like only 100 get synced

2. I'm getting an error "Failed to retrieve a new wallabag auth token, refresh token is expired" at the moment after not syncing for some time. I tried it with a new client key but no luck. Anything I can do to get sync started again?

Thank you very much!

**Comment by hugot** [comment]
Hi, I'm very sorry for the late response! I haven't been on github for a while and for some reason I haven't received email about your issue!

1. Yes that's right. It mostly has to do with not wanting to practically DDOS the wallabag server with 1000's of requests. If I remember correctly, the tool retrieves each article individually to check if it has been read or not, so it has the potential to put some strain on the server.

2. I m no longer using miniflux and wallabag, so I haven't been maintaining this package. The problem could be that wallabag doesn't support the password grant type anymore? I haven't kept up with wallabag's development so I don't know for sure. https://github.com/hugot/n2wal/blob/8ce177a4e6101323263b4d80f4b7dcf2f1cfdbcf/n2wal.el#L54

I'm sorry the tool isn't working anymore, but since I'm no longer using either of the services, I have no plans to continue development/maintenance of this package.

**Comment by KingKarlo** [comment]
Hi, no problem at all, thank you that you've replied anyway!

It looks like we have a similar situation since Miniflux isn't my primary RSS reader anymore too. So your tool is a brilliant help for migrating via wallabag, apparently my use case shifted over the time.

I guess I've found a solution for both issues in the meantime. I was able to adjust the number of articles in the script so I can check more articles at once. For the password issue my workaround is to run through the setup process again and then use the old feed setup. That works pretty well.

Thank you again for your help and the great tool.
